test/metadata/EnvBuilds.test.tsx

```diff
@@ -24,4 +24,29 @@ describe("<EnvBuilds />", () => {
     const progressBar = component.getByRole("progressbar");
     expect(progressBar).toBeInTheDocument();
   });
+
+  it("should render link to log if selected build failed", () => {
```

Comment:
Two things about this are causing lint issues:

1. `BUILD` isn't a valid `Build` class as is, it's missing a `status_info` attribute
2. `EnvBuilds` takes `currentBuildId` and `mode` parameters. I know these aren't involved in what you're testing here, but it might be good to explicitly set them here in part to satisfy the linter, but also for folks looking to the tests as examples of how the component gets used.

Reply:
But I'm lazy and I don't want to 😜
(But I did. I went ahead and fixed and just now pushed the commit.)